public/javascripts/ui/remove-bin.js

```diff
@@ -28,6 +31,8 @@ function showSuccess(message) {
     }
     messageEl.hidden = false;
     messageEl.textContent = message;
+    // can't have the user going back to a bin that doesn't exist.
```

Comment:
This is to prevent existing code from automatically redirecting them to the getBinInfo.html page.